### PR TITLE
Add missing cli methods to the list

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

A few methods do not on the list on the tutorial page.

## Description of the changes

Add methods. Their descriptions come from their API pages.

### CODE REVIEW

**Comment by nzw0301** [comment]
By the way, can we provide method names with links on the first column of the table for usability?

File: tutorial/20_recipes/004_cli.py [hunk 1]
```diff
@@ -17,7 +18,7 @@
     studies, Show a list of studies.
     study optimize, Start optimization of a study.
     study set-user-attr, Set a user attribute to a study.
-    tell, Finish a trial, which was created by the ask command.
+    tell, Finish a trial\\, which was created by the ask command.
```
Comment (nzw0301) [inline tutorial/20_recipes/004_cli.py:21]: Since `,` is used to create the table, I've added `\\` as an escape for the comma

**Review by toshihikoyanase — APPROVED** [review]
Good catch! Thank you!

**Comment by toshihikoyanase** [comment]
> By the way, can we provide method names with links on the first column of the table for usability?

That's a good idea! Can we work on it in a follow-up PR to minimize the changes?

**Comment by nzw0301** [comment]
thank you for your quick review! Adding links might not be simple. Let's address it as a followup.

**Comment by toshihikoyanase** [comment]
That makes sense. Let me merge this PR.

**Pull request merged**